src/es.jl

```diff
@@ -11,13 +11,14 @@
 # Plus-selection: parents are deterministically selected from the set of both the parents and offspring
 #
 function es(  objfun::Function, N::Int;
-              initPopulation::Individual = ones(N),
+              initPopulation::Union{Nothing, Vector} = nothing,
               initStrategy::Strategy = strategy(),
-              recombination::Function = (rs->rs[1]),
-              srecombination::Function = (ss->ss[1]),
-              mutation::Function = ((r,m)->r),
-              smutation::Function = (s->s),
-              termination::Function = (x->false),
+              creation::Function = (n -> rand(n)),
```

Comment:
Now, the `creation` function could  generate vector with the type different from `initPolulation` if only `initPolulation` parameter passed.

Reply:
My intent here is that either a creation function or an initPopulation vector is passed, and if both are passed, the creation function is not called. Unfortunately Julia functions are not parameterised by argument or return types, so we cannot enforce creation (or for that matter, any other function used) to accept or return compatible types.